docs/AdminEndpoints.md

```diff
@@ -0,0 +1,23 @@
+# Admin Endpoints
```

Comment:
This detail looks great. What do you think of structuring it into this format:

```
## `GET /cache/<key>`

### Behavior
Searches through the xds-relay cache and outputs the contents of the cached xDS response for the given aggregated key.

### Parameters
*`<key>`*: The aggregated key or prefix to be searched.
  * Argument supports wildcard (*) glob suffix patterns 
  * ...

### Example
...

Reply:
Added this structure, but made example outputs brief to keep file short